tests/js/sentry-test/initializeOrg.tsx

```diff
@@ -46,13 +60,31 @@ export function initializeOrg({
     },
   ]);
 
+  /**
+   * A collection of router props that are passed to components by react-rotuer
+   *
+   * Pass custom router params like so:
+   * ```ts
+   * initializeOrg({router: {params: {alertId: '123'}}})
+   * ```
+   */
+  const routerProps: RouteComponentProps<RouterParams, {}> = {
+    params: router.params as any,
```

Comment:
```suggestion
    params: router.params as Record<string, any>,
```

🤷‍♀️

Reply:
it just gets overridden above with `RouteComponentProps<RouterParams, {}>`